### PR TITLE
Added recipe for opencensus-ext-azure

### PR DESCRIPTION
**Pull request by krisevans**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/opencensus-ext-azure/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  summary: 'The Azure Monitor Log Handler allows you to export Python logs to Azure Monitor.'
+  description: |
+    The Azure Monitor Log Handler allows you to export Python logs to Azure Monitor.
+  doc_url: https://github.com/census-instrumentation/opencensus-python/tree/master/contrib/opencensus-ext-azure
+  dev_url: https://github.com/census-instrumentation/opencensus-python/tree/master/contrib/opencensus-ext-azure
+
+extra:
+  recipe-maintainers:
+    - krisevans
+    - dhirschfeld
```
Comment (dhirschfeld) [inline recipes/opencensus-ext-azure/meta.yaml:46]: I'm happy to be a maintainer for this recipe

File: recipes/opencensus-ext-azure/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+requirements:
+  build:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - opencensus >=0.8.dev0, <1.0.0
+    - psutil >=5.6.3
+    - requests >=2.19.0
```
Comment (dhirschfeld) [inline recipes/opencensus-ext-azure/meta.yaml:26]: We can use the deps at the specific commit tagged `v0.7.11`:
https://github.com/census-instrumentation/opencensus-python/blob/v0.7.11/contrib/opencensus-ext-azure/setup.py#L41-L45
```suggestion
    - python
    - opencensus >=0.7.11,<1.0.0
    - psutil >=5.6.3
    - requests >=2.19.0
```

File: recipes/opencensus-ext-azure/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - opencensus >=0.7.11,<1.0.0
```
Comment (dhirschfeld) [inline recipes/opencensus-ext-azure/meta.yaml:24]: They've got very unusual versioning for this monorepo - it doesn't follow the versioning of the parent project and they don't tag versions for the sub-projects 😬 Nevertheless, AFAICT the `1.0.6` version should correspond to the `0.7.12` version of the parent project:
```suggestion
    - opencensus >=0.7.12,<1.0.0
```

Comment (dhirschfeld) [reply recipes/opencensus-ext-azure/meta.yaml:24]: ...which means we'll have to wait for the `opencensus-feedstock` PR to be merged.

File: recipes/opencensus-ext-azure/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+  host:
+    - python
```
Comment (dhirschfeld) [inline recipes/opencensus-ext-azure/meta.yaml:20]:
```suggestion
    - python >=3.6
```

File: recipes/opencensus-ext-azure/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+    - pip
+  run:
+    - python
```
Comment (dhirschfeld) [inline recipes/opencensus-ext-azure/meta.yaml:23]:
```suggestion
    - python >=3.6
```

**Comment by krisevans** [comment]
Closing as a feedstock already exists

**Pull request closed without merging**